src/index.js

```diff
@@ -1,3 +1,47 @@
-import './styles/main.scss'
+import 'bootstrap';// eslint-disable-line
+import './styles/main.scss';
 
-console.log('hello world')
\ No newline at end of file
+import AjaxWheather from './js/ajaxfetch';
+import Display from './js/display';
+
+
+(function () { // eslint-disable-line
```

Comment:
- [ ] It uses ES6 syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/)).
Please change this function keyword to an anonymous arrow function.